File: bjj-be/src/main/java/com/jipderksen/bjj/models/RolPartner.java
package com.jipderksen.bjj.models;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * This class <description of functionality>
 *
 * @author jipderksen
 */
@Data
@AllArgsConstructor
public class RolPartner {
    private String name;
    private Belt belt;
    private boolean gender;
}

File: bjj-fe/src/app/models/rol-partner.ts
import {Belt} from './belt';

export class RolPartner {
  name: string;
  belt: Belt;
  gender: boolean;

  constructor(name: string, belt: Belt, gender: boolean) {
    this.name = name;
    this.belt = belt;
    this.gender = gender;
  }
}
